stablehlo/transforms/CMakeLists.txt

```diff
@@ -25,9 +29,12 @@ add_mlir_dialect_library(StablehloPasses
   StablehloRefineShapes.cpp
   VhloLegalizeToStablehlo.cpp
   VhloToVersion.cpp
-
+  ChloLegalizeToStablehlo.cpp
```

Comment:
We'll need `BUILD.bazel` changes as well. I'll send a diff patch you can apply with required bazel changes shortly.

Reply:
Really appreciate everything you do.